api-server/src/events/events.module.ts

```diff
@@ -9,8 +9,11 @@ import { NotificationsModule } from '../notifications/notifications.module';
 import { DatetimeVenueModel } from './models/datetime-venue.model';
 import { EventAdminMetadataModel } from './models/event-admin-metadata.model';
 import { VenueModel } from '../venues/models/venue.model';
+import { PartnerModel } from '../users/models/partner.model';
 import ExistingEventDetectionService from './existing-event-detection-service';
 import { ExistingEventDetectionController } from './existing-event-detection-controller';
+import { UserModel } from '../users/users.modules';
+import UsersService from '../users/users.service';
```

Comment:
more of the aforementioned nest dependency injection plumbing madness, that is probably though possible not still needed :-) Again not too worried about it but may take another pass at a way to make this cleaner